src/directives/until.ts

```diff
@@ -27,6 +27,11 @@ interface AsyncState {
 
 const _state = new WeakMap<Part, AsyncState>();
 
+function isThenable(value: unknown): value is PromiseLike<unknown> {
```

Comment:
This might be extra bytes for nicer type guards. What's the size difference (`npm run checksize`)?

Reply:
8 bytes growth total from this PR. 3218 -> 3226

It looks like this file isn't tracked by that script though, as it's a directive, and that script just tracks the main lit-element.js file.

Reply:
I was wondering if there would be a size reduction in a `isFunction` helper?

Reply:
Maybe? Need to get a size check implemented for all the code, including directives

Reply:
Now just 3 bytes growth: 3221

Reply:
@stramel there's also a speed tradeoff to balance against byte size. An helper that wraps `typeof` requires a function call or the inliner to kick in on some pretty hot code paths.

Reply:
Ah, good point. It will be easier to test the trade-offs once benchmarking and size-checks are both built out.